ranger/ext/img_display.py

```diff
@@ -685,6 +693,8 @@ class KittyImageDisplayer(ImageDisplayer, FileManagerAware):
         https://github.com/kovidgoyal/kitty/blob/master/graphics-protocol.asciidoc"""
     protocol_start = b'\x1b_G'
     protocol_end = b'\x1b\\'
+    answer_end = protocol_end
+    use_placeholder = False
```

Comment:
It's not clear to me what the placeholder is about, is this for testing?

Reply:
https://sw.kovidgoyal.net/kitty/graphics-protocol/#unicode-placeholders

> You can also use a special Unicode character U+10EEEE as a placeholder for an image. This approach is less flexible, but it allows using images inside any host application that supports Unicode and foreground colors (tmux, vim, weechat, etc.) and has a way to pass escape codes through to the underlying terminal.

Because tmux doesn't understand the protocol, the image will not properly render. For example in the case where the user swap two tmux pane, the image will be at the same place.